From 11f68b1c71b4cc14ab601e71d663fa111823760a Mon Sep 17 00:00:00 2001
From: Meng Dongyang <daniel.meng@rock-chips.com>
Date: Wed, 15 Feb 2017 20:53:36 +0800
Subject: [PATCH] Documentation: bindings: add otg-vbus-gpios property for
 Rockchip USB2PHY

Add otg-vbus-gpios optional property to assigned a gpio to
control vbus of otg port.

Change-Id: I257a53edc4d62543f8ac9c7591c29e7231227c20
Signed-off-by: Meng Dongyang <daniel.meng@rock-chips.com>
---
 Documentation/devicetree/bindings/phy/phy-rockchip-inno-usb2.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/phy/phy-rockchip-inno-usb2.txt b/Documentation/devicetree/bindings/phy/phy-rockchip-inno-usb2.txt
index e3ec8bd5e592..3a16dee132cc 100644
--- a/Documentation/devicetree/bindings/phy/phy-rockchip-inno-usb2.txt
+++ b/Documentation/devicetree/bindings/phy/phy-rockchip-inno-usb2.txt
@@ -18,6 +18,7 @@ Optional properties:
 		 Refer to clk/clock-bindings.txt for generic clock
 		 consumer properties.
  - rockchip,u2phy-tuning; when set, tuning u2phy to improve usb2 SI.
+ - otg-vbus-gpios : assign the gpio to control vbus of otg port.
 
 Required nodes : a sub-node is required for each port the phy provides.
 		 The sub-node name is used to identify host or otg port,
-- 
2.35.3

